src/test/scala/edu/ie3/simona/scheduler/SchedulerSpec.scala

```diff
@@ -103,6 +105,49 @@ class SchedulerSpec
       parent.expectMessage(Completion(schedulerActivation))
     }
 
+    "scheduling with parent when earliest tick changes" in {
```

Comment:
This sounds and looks very similar to test at line 165 in PhaseSwitchSchedulerSpec.

Reply:
Ok fine for me. Just wanted to check if you see changes to merge both.